quodlibet/docs/guide/searching.rst

```diff
@@ -203,3 +203,19 @@ name ``"Sigur Rós"``.
 
 
 Now you can search anything!
+
+
+Reusing queries
+---------------
+
+Complex queries can be splitted into simpler ones.  Also, a query can be reused 
+in other ones.  In this way it is easier to change and administer your searches.
+
+In order to do so, the ``Include Saved Search`` plugin must be activated.  
+If you create a saved search named ``Unrated`` you can search for unrated songs from the Beatles
+like this:
+
+``@(saved: Unrated) AND Beatles``
```

Comment:
That's true. I'm using other search engines, and I have confused the syntax.  Sorry